Venmo/src/main/java/com/braintreepayments/api/VenmoClient.java

```diff
@@ -163,10 +169,12 @@ public void onResult(@Nullable final Configuration configuration, @Nullable Exce
                 }
 
                 String exceptionMessage = null;
-                if (!configuration.isVenmoEnabled()) {
-                    exceptionMessage = "Venmo is not enabled";
-                } else if (!deviceInspector.isVenmoAppSwitchAvailable(activity)) {
-                    exceptionMessage = "Venmo is not installed";
+                if (!request.getFallbackToWeb()) {
+                    if (!configuration.isVenmoEnabled()) {
```

Comment:
The `enabled` check for the merchant account should happen even if `fallBackToWeb` is true right?

Reply:
You're right, update: https://github.com/braintree/braintree_android/pull/911/commits/86ce5513e20bae65950237b13d8f42fce49024d1